cdap-kubernetes/src/main/java/io/cdap/cdap/k8s/runtime/KubeTwillRunnerService.java

```diff
@@ -140,6 +140,14 @@ public KubeTwillRunnerService(MasterEnvironmentContext masterEnvContext,
     this.liveInfoLock = new ReentrantLock();
     this.jobCleanupIntervalMins = jobCleanupIntervalMins;
     this.jobCleanBatchSize = jobCleanBatchSize;
+
+    try {
```

Comment:
it's generally not good to do anything that can throw an exception in the constructor, and it seems kind of hacky to avoid starting this service.

If possible, it would be cleaner if we could configure this to skip monitoring through a config setting in MasterEnvironmentContext.getConfigurations() (which is just the map version of the cconf). It's not clear to me where the MasterEnvironment is getting set, but if there is a way to add a property into the cconf before that happens, I think that would be better.

Reply:
Done. Added a new configuration that specifies which Runnables that need disabled KubeTwillRunnerService.